src/ignitus-UserInterfaceBook/Components/Organisms/interfaceMessageSettings/Components/index.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { useState } from 'react';
+import { MessageSettings, Heading2 } from '../../../../../ignitus-Shared';
+import { Interface } from '../../../../styles';
+
+export const InterfaceMessageSettings: React.FC = () => {
+  const [anyone, setAnyone] = useState(false);
+  const [connections, setConnections] = useState(false);
+  const [directConnections, setDirectConnections] = useState(false);
+
```

Comment:
We won't be needing this after you use the already implemented `DefaultCheckbox` component.